### PR TITLE
docs: fleshed out privacy protection (people and screens)

### PR DESCRIPTION
**Pull request by ericleehy**
For your review and comments, please.

### CODE REVIEW

File: docs/source/use_cases/index.rst [hunk 1]
```diff
@@ -113,6 +120,10 @@ Privacy Protection
 +--------------------------------------------------------+-----------------------------+
 | |privacy_protection_faces_gif|                         | |privacy_protection_lp_gif| |
 +--------------------------------------------------------+-----------------------------+
+| |privacy_protection_people_screens_doc|                |                             |
++--------------------------------------------------------+-----------------------------+
+| |privacy_protection_people_screens_gif|                |                             |
++--------------------------------------------------------+-----------------------------+
```
Comment (eric-kwok-nt) [inline docs/source/use_cases/index.rst]: Perhaps can remove the row separator in the second column to be consistent with other use cases

```suggestion
+--------------------------------------------------------+-----------------------------+
| |privacy_protection_people_screens_doc|                |                             |
+--------------------------------------------------------+                             +
| |privacy_protection_people_screens_gif|                |                             |
+--------------------------------------------------------+-----------------------------+
```

File: docs/source/use_cases/privacy_protection_people_screens.rst [hunk 2]
```diff
@@ -41,24 +39,24 @@ PeekingDuck with the configuration file |pipeline_config|_ as shown:
...
-except that it only has a single channel and each pixel on the mask is a binary of either 1 or 0,
-which indicates whether a specific class of thing is present (1) or absent (0) in a particular
-location of the image. For more information on how to adjust the <insert-model-name> node, check
-out the <insert-link-to-model-page>.
+We use an open source instance segmentation model known as `Mask R-CNN <https://arxiv.org/abs/1703.06870>`_
+to obtain the masks of persons, computer screens and laptops. The masks are akin to the input frames
+or images, except that it only has a single channel and each pixel on the mask is a binary of either
+1 or 0, which indicates whether a specific class of thing is present (1) or absent (0) in a
+particular location of the image. For more information on how to adjust the Mask_RCNN node, check
+out the :doc:`Mask_RCNN configurable parameters </nodes/model.mask_rcnn>`.
```
Comment (eric-kwok-nt) [inline docs/source/use_cases/privacy_protection_people_screens.rst]: Would it be better to change to the following instead, following the `zone_counting.rst` convention?

```suggestion
particular location of the image. For more information on how to adjust the ``mask_rcnn`` node, check
out its :doc:`configurable parameters </nodes/model.mask_rcnn>`.
```

**Review by eric-kwok-nt — COMMENTED** [review]
Just some minor comments, otherwise LGTM.

File: docs/source/use_cases/index.rst (hunk 1 above)
Comment (ericleehy) [reply docs/source/use_cases/index.rst]: Good catch. Amended.

File: docs/source/use_cases/privacy_protection_people_screens.rst (hunk 2 above)
Comment (ericleehy) [reply docs/source/use_cases/privacy_protection_people_screens.rst]: FYI, I had followed the copy from privacy_protection_faces.rst.

In any case, I have implemented your suggested amendment.

**Review by eric-kwok-nt — APPROVED** [review]

**Pull request merged**